### PR TITLE
Improve sections in example

### PR DESCRIPTION
**Pull request by wreise**

### CODE REVIEW

**Comment by wreise** [comment]
@ulupo?

**Comment by ulupo** [comment]
I wasn't notified of this somehow...

File: doc/notebooks/examples.rst [hunk 1]
```diff
@@ -4,29 +4,28 @@ Examples
...
-   :hidden:
+.. toctree::
+   :maxdepth: 1
 
    classifying_shapes
    lorenz_attractor
    voids_on_the_plane
 
-.. include:: lorenz_attractor.rst
+..
```
Comment (ulupo) [inline doc/notebooks/examples.rst:14]: Is this commented out?

**Review by ulupo — APPROVED** [review]

File: doc/notebooks/examples.rst (hunk 1 above)
Comment (wreise) [reply doc/notebooks/examples.rst:14]: yes!

**Pull request merged**